qemu/tests/cfg/vdi_image_create.cfg

```diff
@@ -0,0 +1,12 @@
+- vdi_image_create:
+    only Host_RHEL.m9
```

Comment:
what if vdi is supported after m9?
do we need to use 'no' instead of 'only' here?